libs/test_positivity.py

```diff
@@ -115,19 +164,23 @@ def run(
             .reindex(columns=input_date_range)
             .rename_axis(columns=CommonFields.DATE)
         )
-        # This calculates the difference only when the cumulative value is a real value `diff_days` apart.
-        # It looks like our input data has few or no holes so this works well enough.
-        diff_df = input_wide.diff(periods=diff_days, axis=1)
 
         methods_with_data = AllMethods._methods_with_columns_available(
-            methods, diff_df.index.get_level_values(PdFields.VARIABLE).unique()
+            methods, input_wide.index.get_level_values(PdFields.VARIABLE).unique()
```

Comment:
I tweaked this so we check for column availability on `df` instead of `diff_df`...  since we don't care about `diff_df` in the pass-thru method.  It would have worked as-is, but this seems more straightforward.  Now `diff_df` is passed into `calculate()` in case it needs it but it's not used for any other logic.